### PR TITLE
Notify third-party admins

### PR DESCRIPTION
**Pull request by kaapstorm**
##### SUMMARY
Adds ability for MOTECH to notify admins of third-party systems when errors occur.

This PR adds only one instance of such a notification. More to follow in subsequent PRs.

##### FEATURE FLAG
"OpenMRS integration" or "DHIS2 Integration"

### CODE REVIEW

File: corehq/motech/openmrs/static/openmrs/js/openmrs_importers.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ hqDefine('openmrs/js/openmrs_importers', [
         // using the Django form field names.
         self.username = ko.observable(properties["username"]);
         self.password = ko.observable(properties["password"]);
+        self.notify_email = ko.observable(properties["notify_email"]);
```
Comment (emord) [inline corehq/motech/openmrs/static/openmrs/js/openmrs_importers.js]: I think that the new standard is to use hungarian notation https://github.com/dimagi/commcare-hq/pull/25394/files

File: corehq/motech/requests.py [hunk 2]
```diff
@@ -48,12 +53,33 @@ class Requests(object):
...
+        """
+        Initialise instance
+
+        :param domain_name: Domain to store logs under
+        :param base_url: Remote API base URL
+        :param username: Remote API username
+        :param password: Remote API plaintext password
+        :param verify: Verify SSL certificate?
+        :param notify_email: A list, or a comma- or space-separated
+            string of email addresses to notify of errors.
```
Comment (emord) [inline corehq/motech/requests.py]: Why is this so flexible? Would it be easier to change `notify_email` to a `StringListProperty` and always pass in a list and error on any other inputs?

**Review by emord — COMMENTED** [review]
I think that the isort comments are wrong, which makes me think that stickler is not picking up our configuration properly (likely that its not considering `dimagi.utils`, `pillowtop`, etc to be a separate class of import)

**Comment by kaapstorm** [comment]
Closing in favor of follow-up PR #25648

**Pull request closed without merging**